refactor(common): fix exceptions doc/var names

saulotoledo opened the pull request:
Fix incorrect documentation strings for HTTP exceptions.

Closes #4042

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: incorrect documentation
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #4042


## What is the new behavior?
Documentation strings are now correct.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

## Other information
If this is merged before #4045, the documentation strings for the exceptions must be updated by #4045.
If this is merged after #4045, this PR should be updated to fix the documentation strings.

kamilmysliwiec commented:
cc @johnbiundo can you take a look?

johnbiundo commented:
Also, regarding related changes:

1) @kamilmysliwiec, we'll need to update the docs accordingly
2) @kamilmysliwiec , since #4045 is a breaking change, I assume you'll not want to merge that one before 7.0, if at all.

saulotoledo commented:
Hi @johnbiundo, thanks for the review!
I updated the PR according to your recommendations. I also changed the "usage notes" at `packages/common/exceptions/http.exception.ts`. Could you check the PR again (especially the last change in the file `http.exception.ts`)?

@johnbiundo, @kamilmysliwiec, after this PR is merged, I will update #4045 (the parameters in the "usage notes" will change after that PR).

johnbiundo commented:
Looks good!  Thanks for doing the dirty work on updating each one!

kamilmysliwiec commented:
> @kamilmysliwiec , since #4045 is a breaking change, I assume you'll not want to merge that one before 7.0, if at all.

@johnbiundo we won't merge #4045 until the next major release (v7.0.0).

Should we also create another PR which will reflect changes made in the #4045?

saulotoledo commented:
@kamilmysliwiec I can just update #4045, or I can create a separated PR if you prefer.
In #4045 we could also decide if you prefer to keep `message`(as I did) or `error` as suggested by others. Just let me know and I will do the proper updates.

kamilmysliwiec commented:
@saulotoledo can you please rebase to 7.0.0 branch and update comments to use the `message` key instead? (just merged your other PR)

kamilmysliwiec commented:
ping @saulotoledo 🏓

saulotoledo commented:
@kamilmysliwiec sorry for the late reply and thanks for the reminder. I am traveling but I am able to do it today in a couple of hours.

kamilmysliwiec commented:
Thank you!

saulotoledo commented:
@kamilmysliwiec I updated the PR. I changed the commit message from "docs" to "refactor" because I updated the variable names according to your changes at https://github.com/nestjs/nest/commit/dece70c3195bbd3aa2a02fc14cecd4fb60b63302#diff-7df9268bccefc4018a65350f53ddcc5c

I also changed the target branch to 7.0.0. Thank you again and sorry for the delay.

kamilmysliwiec commented:
Thank you @saulotoledo!
